oat-sa/tao-core-ui-fe
# Fix/ADF-1191/Double slash in criteria

### Related to: https://oat-sa.atlassian.net/browse/ADF-1191

### Summary

Fix the visual glitch in the list of criteria: a double slash was added to each duplicated property.

### Details

Remove the duplicated slash.
Also, fix a bad design with inlined HTML templates.

### How to test
-  compile the code
    ```sh
    npm run build:all
    ```
-  run the unit tests
    ```sh
    npm run test:keepAlive
    ```
    Open the browser at: http://127.0.0.1:5400/test/searchModal/advancedSearch/test.html
    Play with the criteria. No more double slashes should show up.
- checkout the branch in `tao/views/node_module/@oat-sa/tao-core-ui`, and check the advanced search in TAO (you will need to also checkout the integration branches in the other repositories)
